misk-jooq/src/test/resources/db-migrations/v0001__create_balance.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE balance(
+    id BIGINT NOT NULL AUTO_INCREMENT primary key,
+    entity VARCHAR(191) NOT NULL,
+    jurisdiction VARCHAR(3) NOT NULL,
+    amount_cents BIGINT NOT NULL,
+    currency VARCHAR(3) NOT NULL,
+    version INT NOT NULL,
+    created_at TIMESTAMP,
+    updated_at TIMESTAMP,
+    UNIQUE KEY idx_float_entity(entity, jurisdiction)
+)
```

Comment:
Prefer to have tests using non-financial domains to make very clear that the tests are fictitious. For instance, misk-hibernate uses a Movie database.